src/components/Common.js

```diff
@@ -51,7 +62,7 @@ export const DetailContainer = styled.div`
   flex-wrap: wrap;
   width: 100%;
 
-  @media (max-width: 1200px) {
+  ${maxWidthMediaQueries("desktop")} {
```

Comment:
```suggestion
  ${maxWidthMediaQueries('desktop')} {
```
sorry

Reply:
no u good! i also updated for all the other instances of `maxWidthMediaQueries` in my newest commit , ty